### PR TITLE
SJSU-1016, Implement Confirmation Page

### PR DESCRIPTION
**Pull request by ynoe**
Confirmation Page Utopia, no more eyesore from staring at an auto parts invoice! Satisfaction Guaranteed!!
To cook up this stew, I created a booking folder and mixed 1/3 tablespoon each of BookingBreakdownTable.js, Confirmation.js and BookingDetailsCard.js. Not forgetting a dash of salt 😉
To spice things up, I went to the spice market and added them to the stew…mmm enjoy~

To adjust to your liking, navigate to the root and front_end directories to do an npm install. 

image

### CODE REVIEW

File: front_end/src/components/booking/BookingBreakdownTable.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  head: {
+    backgroundColor: theme.palette.common.black,
+    color: theme.palette.common.white,
+    fontSize: 14
+  },
+  body: {
+    fontSize: 14
+  },
+  footer: {
+    backgroundColor: "#84ffff",
```
Comment (SandroSal) [inline front_end/src/components/booking/BookingBreakdownTable.js]: can you use our secondary color? #a0cdfd

File: front_end/src/components/booking/BookingDetailsCard.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        </Typography>
+        <Typography className={classes.pos} color="textSecondary">
+          Booking Date: 2/2/2019
+        </Typography>
+        <Typography component="p">
+          Room Type: Double room
+          <br />
+          {"Check In: 4/2/2019"}
+          <br />
+          {"Check Out: 4/6/2019"}
```
Comment (SandroSal) [inline front_end/src/components/booking/BookingDetailsCard.js]: Please add  "Number of Nights"

File: front_end/src/components/booking/BookingBreakdownTable.js (hunk 1 above)
Comment (ynoe) [reply front_end/src/components/booking/BookingBreakdownTable.js]: changes made

File: front_end/src/components/booking/BookingDetailsCard.js (hunk 2 above)
Comment (ynoe) [reply front_end/src/components/booking/BookingDetailsCard.js]: changes made

File: front_end/src/components/booking/BookingDetailsCard.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        <Typography variant="h5" component="h2">
+          Confirmation Number: 123241
+        </Typography>
+        <Typography className={classes.pos} color="textSecondary">
+          Booking Date: 2/2/2019
+        </Typography>
+        <Typography component="p">
+          Room Type: Double room
+          <br />
+          {"Number of Nights: 4"}
```
Comment (ynoe) [inline front_end/src/components/booking/BookingDetailsCard.js]: here

**Pull request merged**